kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/activity/ConversationActivity.java

```diff
@@ -1221,7 +1220,14 @@ public void showVideoCapture() {
     }
 
     public boolean isServiceDisconnected() {
-        return false;
+        boolean isDebuggable = (0 != (getApplicationInfo().flags & ApplicationInfo.FLAG_DEBUGGABLE));
+        boolean disconnect = alCustomizationSettings != null && !alCustomizationSettings.isAgentApp()
+                && MobiComUserPreference.getInstance(this).getPricingPackage() == PackageType.STARTUP.getValue()
+                && !isDebuggable;
```

Comment:
We can just check if the role is `agent` then don't disable instead of `isAgentApp()`. I think SDK shouldn't know if it's inside an agent app or not.